config/initializers/acts_as_taggable_on.rb

```diff
@@ -14,9 +14,9 @@
   #accepts_nested_attributes_for :tag_languages, allow_destroy: true
   scope :translated_in_current_language_and_no_translation, -> (lang_str) { ActsAsTaggableOn::Tag.includes(:locale_languages)
                                                                 .where(locale_languages: { id: nil })
-                                                                .or(ActsAsTaggableOn::Tag.includes(:locale_languages)
+                                                                .or( ActsAsTaggableOn::Tag.includes(:locale_languages)
```

Comment:
I think it's better readable if you write: OR (capital letters), because it's a logical operator that connects the two statements (that means it's not the same level as the other two)